api/Document.json

```
@@ -4671,7 +4695,7 @@
             }
           },
           "status": {
-            "experimental": false,
+            "experimental": true,
```

Would you mind reverting this? Since there's two implementations (Chrome and Safari), we can let this be.

```suggestion
            "experimental": false,
```